tests/vmi_configuration_test.go

```diff
@@ -2110,4 +2110,21 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	It("Masquerede VMI with Guest agent should not update VMI IP", func() {
+		vmi := tests.NewRandomFedoraVMIWitGuestAgent()
```

Comment:
/Wit/With/ ?

Reply:
That's the typo in the utils itself
https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L1913
Will do a separate PR to fix that